src/app/layout.config.tsx

```
@@ -1,33 +1,58 @@
+export const linkItems: LinkItemType[] = [
+  {
+    text: 'Documentation',
+    url: '/docs',
+    active: 'nested-url',
+  },
+  {
+    type: 'icon',
+    url: 'https://github.com/techwithanirudh/matsu-docs',
+    text: 'Github',
+    icon: (
+      <svg role="img" viewBox="0 0 24 24" fill="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)